src/angular-gridster.js

```diff
@@ -1355,6 +1356,11 @@
 					if ($target.hasClass('gridster-item-resizable-handler')) {
 						return false;
 					}
+					
+					//checking if user clciked on widgets dragged area
+					if (gridster.draggable.element && (!(e.target.classList.contains(gridster.draggable.element) || $(e.target).parents('.' + gridster.draggable.element).length))) {
```

Comment:
Can you check if the `parents` method is available if jQuery is not loaded?  If it isn't, then you might have to add `&& $.fn.parents` to avoid errors for pure Angular use cases.